restli-common/src/main/pegasus/com/linkedin/restli/common/ExtensionSchemaAnnotation.pdsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "ExtensionSchemaAnnotation",
+  "namespace": "com.linkedin.restli.common",
+  "doc": "Specifies the extension schema annotation configuration for defining the entity relationship among entities.",
+  "fields": [
+    {
+      "name": "resource",
+      "type": "string",
+      "doc": "Specifies resource name to avoid the ambiguity in multi-version scenario."
```

Comment:
Should this be optional then?

Reply:
As we discussed with GraphQL before, we make the "resource" field as required to avoid the ambiguity if there is multi-version use case.